tests/cloud_playbooks/upload-logs-gcs.yml

```diff
@@ -49,7 +49,7 @@
         CLOUDSDK_CORE_DISABLE_PROMPTS: 1
         CLOUDSDK_INSTALL_DIR: "{{dir}}"
       no_log: True
-      ignore_errors: true
+      failed_when: false
```

Comment:
This might be one I want to set to ignore_errors instead, but I'm ok if you feel this is ok